sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/routing/PartitionKeyInternal.java

```diff
@@ -170,6 +170,21 @@ public boolean equals(Object obj) {
         return this.compareTo(pki) == 0;
     }
 
```

Comment:
Are we tracking issues? Should there not be a reference to an issue when it comes to things like this that--I presume--might need to be done before GA?

Reply:
yes, its referenced in the comment. https://github.com/Azure/azure-sdk-for-java/issues/9046 (

Reply:
thanks for the issue reference